src/WebJobs.Script.WebHost/Security/ScriptSecrets.cs

```diff
@@ -23,6 +24,9 @@ protected ScriptSecrets()
         [JsonIgnore]
         public abstract ScriptSecretsType SecretsType { get; }
 
+        [JsonProperty(PropertyName = "metadata")]
+        public virtual SecretsMetadata Metadata { get; }
```

Comment:
Do we really need a structured object for this, or would a simple GeneratedBy property suffice? That's what we have in our VS generated function.json files.